volcano-device-plugin.yml

```diff
@@ -83,6 +83,7 @@ spec:
       serviceAccount: volcano-device-plugin
       containers:
       - image: volcanosh/volcano-device-plugin:latest
```

Comment:
One last thing: l think this official image will not update due to this repo doesn't have ci. 
Need someone with authority to do this, @william-wang @Thor-wl Any Idea?

Reply:
> One last thing: l think this official image will not update due to this repo doesn't have ci.
> Need someone with authority to do this, @william-wang @Thor-wl Any Idea?

Let me update it.
/assign